src/tree/utils.js

```diff
@@ -1,20 +1,14 @@
-import _ from 'lodash';
-
 const MULTI_MODULE_REGEXP = /^multi /u;
 
 export function getModulePathParts(moduleData) {
   if (MULTI_MODULE_REGEXP.test(moduleData.identifier)) {
     return [moduleData.identifier];
   }
 
-  const parsedPath = _
-  // Removing loaders from module path: they're joined by `!` and the last part is a raw module path
```

Comment:
These comments are important as they document the reason why the lines of code exist in the first place. Could we keep them?

Reply:
Sure, will do!